driver/main.c

```diff
@@ -1871,25 +2034,39 @@ static int record_event_consumer(struct ppm_consumer_t *consumer,
 		if (cbres == PPM_SUCCESS) {
 			ASSERT(freespace < sizeof(struct ppm_evt_hdr) + args.arg_data_offset);
 			ring_info->n_drops_buffer++;
+			drops_buffer_syscall_categories_counters(event_type, ring_info);
 		} else if (cbres == PPM_FAILURE_INVALID_USER_MEMORY) {
```

Comment:
Just noticed now that this is not so correct...If the event parameters are different from the value in the table we drop the event with `n_drops_buffer++`. Even if this should never happen it is not so clear, maybe we need just to remove this without incrementing the drop_buffer or add a new category of drops... BTW this doesn't regard this PR, I wrote here just to remember it :)

Reply:
Wondered about that too. Will definitely remove the new added line.